packages/rx_bloc_cli/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## [4.1.0]
```

Comment:
Let's deliver this right after the breaking change. With this being said let's target 5.1.0

Reply:
Will do